lib/editor/containers/ActiveEditorLockManager.js

```diff
@@ -0,0 +1,118 @@
+// @flow
+
+import { Component } from 'react'
+import { connect } from 'react-redux'
+
+import * as editorActions from '../actions/editor'
+import type { AppState, StatusState } from '../../types/reducers'
+
+type Props = {
+  checkLockStatus: typeof editorActions.checkLockStatus,
+  disableLock?: boolean,
+  errorStatus: StatusState,
+  // feedIsLocked: boolean,
+  feedSourceId: string,
+  itemToLock: string,
+  lockEditorFeedSource: typeof editorActions.lockEditorFeedSource,
+  // lockEditorFeedSourceIfNeeded: typeof editorActions.lockEditorFeedSourceIfNeeded,
+  // removeEditorLock: typeof editorActions.removeEditorLock,
```

Comment:
These can probably go! (there's a few others in this file too)

Reply:
Yes, removed in https://github.com/ibi-group/datatools-ui/pull/852/commits/da9d219113c3b89f02471f3f9ab1becc0b1e2fa7.